benchmarks/advection_in_annulus/advection_in_annulus.prm

```diff
@@ -0,0 +1,133 @@
+# on the overall cooling of the model.
+# 
+# See run_all_models.sh for running multiple iterations
+# of various parameters in a row.
+#
+# Also see creates_formatted_tables for grabbing specific
+# data points from the log file.
+
+############### Global parameters
+# We use a 2d setup. We are only interested in
+# advection, so we prescribe a Stokes flow and
+# solve only for advection.
+
+set Additional shared libraries            = ./libadvection_in_annulus.so
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = single Advection, no Stokes
+set Output directory                       = output_advection_in_annulus
+set Pressure normalization                 = surface
+set Timing output frequency                = 100
+
+subsection Prescribed Stokes solution
+  set Model name = advection in annulus
+end
+
+############### Parameters describing the model
+# We are interested in the flow of heat out of the top
+# boundary and how the advection is stabilized. Interesting
+# values to look at are in subsection Discretization, which
+# control the stabilization directly, as well as thermal
+# conductivity which effectively controls the Rayleigh number.
+# The thermal conductivity and Rayleigh number are inversely
+# proportional.
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  # NOTE: the analytical solutions are hard-coded
+  # to these geometry values: do not change them.
+  subsection Spherical shell
+    set Inner radius = 1 
+    set Outer radius = 2
+    set Opening angle = 360 
+  end 
+end
+
+subsection Discretization
+  set Temperature polynomial degree = 2
+
+  subsection Stabilization parameters
+    set beta = 0.052
+    set cR = 0.11
+  end
+end
+
+subsection Material model
+  set Model name = simpler
+
+  subsection Simpler model
+    set Reference density = 1.0
+    set Reference temperature = 1.0
+    set Viscosity = 1.0
+    set Thermal conductivity = 1e-3
+    set Reference specific heat = 1.0
+    set Thermal expansion coefficient = 0
+  end
+end
+
+subsection Gravity model
+  set Model name = radial constant
+
+  subsection Radial constant
+    set Magnitude =  1
+  end
+end
+
+############### Parameters describing the temperature field
+# Since we are interested in the flow of heat through
+# the top of the model, we set the temperture to 1
+# everywhere within the model except for the top
+# boundary, which is set to 0.
+
+subsection Boundary temperature model
+  set List of model names = function
+
+  subsection Function
+    set Function expression = 0
+  end
+
+  set Fixed temperature boundary indicators = top
+end
+
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Function expression = 1
+  end
+end
+
+
+############### Parameters describing the discretization
```

Comment:
Please remove this paragraph of comments, it is not really relevant for this benchmark.